cloud/deploys/dev_azure/deploy

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env bash
```

Comment:
I think dev_azure needs a .sh file and we source that when deploy instead of having a specific deploy file